lua/entities/acf_base_scalable/init.lua

```diff
@@ -61,18 +61,37 @@ end ---------------------------------------------
 do -- Entity linking and unlinking --------------
 	local LinkText   = "%s can't be linked to %s."
 	local UnlinkText = "%s can't be unlinked from %s."
+	local timer   = timer
 
-	function ENT:Link(Target)
+	function ENT:Link(Target, FromChip)
 		if not IsValid(Target) then return false, "Attempted to link an invalid entity." end
 		if self == Target then return false, "Can't link an entity to itself." end
 
 		local Class    = Target:GetClass()
-		local Function = ACF.GetClassLink(self:GetClass(), Class)
+		local LinkData, Reversed = ACF.GetClassLink(self:GetClass(), Class)
+
+		if LinkData == nil then return false, "Links between these two entities are impossible" end
```

Comment:
If `LinkData` never actually receives a `false` value as a valid result, then you could just change this condition to `if not LinkData`